### ISSUE TITLE
Bot framework integration with Cisco Webex teams bot

### ISSUE THREAD

**Issue by Bradrajkumar** [microsoft/BotFramework-Services#215]
Hi all, 
The description provided for webex as a channel is not sufficient to fully integrate the bot framework with Webex bot. Where I can get fully described information for bot framework integration with Cisco Webex teams bot.

**Comment by dmvtech** [comment]
Hi @Bradrajkumar By _description_, are you referring to this documentation?:
https://docs.microsoft.com/en-us/azure/bot-service/bot-service-adapter-connect-webex?view=azure-bot-service-4.0

**Comment by Bradrajkumar** [comment]
hi @dmvtech ,
yes i am using the same link which you have added here.

**Comment by dmvtech** [comment]
Thank you. Investigating. I'll let you know what I find.

**Comment by dmvtech** [comment]
Hi @Bradrajkumar, did you make sure your both was running per [this section](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-adapter-connect-webex?view=azure-bot-service-4.0#create-and-update-a-webex-webhook)?. 

>  To do this your bot must be running, so that Webex can verify the URL to the endpoint is valid.

**Comment by Bradrajkumar** [comment]
Yes , I did this part but I haven't published but I am running it locally.
There is a "put" method recommended for connection but it's not worth it
using and instead I used "post" and it's working ok for establishing the
connection.
